sources/api/controllers/GroupController.ts

```diff
@@ -64,22 +76,24 @@ async function getUser(userRepository: Repository<User>, pseudo: string) : Promi
 //
 export async function groupInfo(req: Request, res: Response) {
     const requestingUser = getRequestingUser(req);
-    const groupId = req.params.group_id;
+
+    const groupId: number = req.params.group_id;
 
     const conn = await DbConnection.getConnection();
-    const groupToUserRepository = conn.getRepository(GroupsToUsers);
-    const groupMember = await GroupsToUsers.getRelation(groupToUserRepository, groupId, requestingUser.id);
-    if (!groupMember) {
+
+    // Get the requested Group
+    const group = await Group.findById(conn, groupId);
+    if (!group) {
         throw new ApiException(403, "Group not accessible");
```

Comment:
Didn't we say we would return 404 in both cases : not found / not accessible ? That's what we did on calendars

Reply:
We will return 403 everywhere !

Reply:
ok !